src/helpers/limit_results.js

```diff
@@ -0,0 +1,8 @@
+const limitResults = (results) => {
+  while (results.length != 9 && results.length > 9) {
```

Comment:
why limit? 
and my first question was more technical, why you ask if its >9 and !=9 ?